mobile/src/main/java/ch/epfl/sdp/mobile/ui/i18n/English.kt

```diff
@@ -16,6 +16,16 @@ object English : LocalizedStrings {
   override val authenticateErrorFailure = "Something went wrong when authenticating"
   override val authenticateOr = "or"
 
+  override val boardColorBlack = "black"
+  override val boardColorWhite = "white"
+  override val boardPieceKing = "king"
+  override val boardPieceQueen = "queen"
+  override val boardPieceRook = "rook"
+  override val boardPieceBishop = "bishop"
+  override val boardPieceKnight = "knight"
+  override val boardPiecePawn = "pawn"
+  override val boardPieceContentDescription = { color: String, rank: String -> "$color $rank" }
```

Comment:
Let's try not to move our strings definitions too often 😄

Reply:
My bad it's just that I didn't resolve correctly the merge conflicts so I had to add them manually .